pkg/image/api/types.go

```diff
@@ -106,6 +128,32 @@ type TagEvent struct {
 	DockerImageReference string
 	// The image
 	Image string
+	// Generation is the spec tag generation that resulted in this tag being updated
+	Generation int64
```

Comment:
Added a comment with some more detail, but basically because it's not needed (generation 0 means no generation, whereas for spec old clients will send nil and new clients need a way to send a value to indicate reset, which will be zero).